File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/ChangelogVO.kt
package com.sdkwork.app

data class ChangelogVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val version: String? = null,
    val releaseDate: String? = null,
    val title: String? = null,
    val features: List<String>? = null,
    val bugFixes: List<String>? = null,
    val improvements: List<String>? = null,
    val status: String? = null,
    val downloadUrl: String? = null
)

File: sdkwork-app-sdk-typescript/src/types/changelog-vo.ts
/** 更新日志响应 */
export interface ChangelogVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 版本号 */
  version?: string;
  /** 发布日期 */
  releaseDate?: string;
  /** 更新标题 */
  title?: string;
  /** 新特性 */
  features?: string[];
  /** Bug修复 */
  bugFixes?: string[];
  /** 改进 */
  improvements?: string[];
  /** 状态 */
  status?: string;
  /** 下载链接 */
  downloadUrl?: string;
}
